src/pages/CytAssist.tsx

```diff
@@ -0,0 +1,337 @@
+                onChangeBarcode('');
+              } else {
+                setPreBarcodeValidationError('');
+                onChangeBarcode(value);
+              }
+            }
+          }}
+          allowEmptyValue={true}
+          placeholder={'V42A20-3752023-10-20'}
+        />
+        {preBarcodeValidationError.length > 0 ? (
+          <MutedText className={'text-red-400'}>{preBarcodeValidationError}</MutedText>
+        ) : preBarcode === undefined || preBarcode.length <= 0 ? (
+          <MutedText>Input barcode and press 'Enter' key</MutedText>
+        ) : (
+          ''
+        )}
+      </div>
+      <div>
+        <Label name={'Labware Type'} />
+        <Select
+          onChange={(e) => {
+            onChangeLabwareType(e.currentTarget.value);
+          }}
+          emptyOption={false}
+          data-testid="output-labware-type"
+        >
+          {objectKeys(LabwareTypeName)
+            .filter(
+              (key) =>
+                LabwareTypeName[key] === LabwareTypeName.VISIUM_LP_CYTASSIST ||
+                LabwareTypeName[key] === LabwareTypeName.VISIUM_LP_CYTASSIST_XL
+            )
+            .map((key) => (
+              <option key={key} value={LabwareTypeName[key]}>
+                {LabwareTypeName[key]}
+              </option>
+            ))}
+        </Select>
+      </div>
+      <div>
+        <Label name={'Slide costings'} />
+        <Select
+          onChange={(e) => {
+            onChangeCosting(e.currentTarget.value);
+          }}
+          emptyOption={true}
+          data-testid="output-labware-costing"
+        >
+          {objectKeys(SlideCosting).map((key) => (
+            <option key={key} value={SlideCosting[key]}>
+              {SlideCosting[key]}
+            </option>
+          ))}
+        </Select>
+      </div>
+    </div>
+  );
+};
+
+/**Table component to display all mappings made between slots in input and output labware**/
+const SlotMappingContentTable = ({ slotCopyContent }: { slotCopyContent: SlotCopyContent[] }) => {
+  return (
+    <Table data-testid="mapped_table">
+      <TableHead>
+        <tr>
+          <TableHeader>Source Barcode</TableHeader>
+          <TableHeader>Source Address</TableHeader>
+          <TableHeader>Destination Address</TableHeader>
+        </tr>
+      </TableHead>
+
+      <TableBody>
+        {slotCopyContent.map((slot) => (
+          <tr key={`${slot.sourceAddress}-${slot.sourceBarcode}`}>
+            <TableCell>{slot.sourceBarcode}</TableCell>
+            <TableCell>{slot.sourceAddress}</TableCell>
+            <TableCell>{slot.destinationAddress}</TableCell>
+          </tr>
+        ))}
+      </TableBody>
+    </Table>
+  );
+};
+
+const CytAssist = () => {
+  const [current, send] = useMachine(() =>
+    slotCopyMachine.withContext({
+      workNumber: '',
+      operationType: 'CytAssist',
+      outputLabwareType: LabwareTypeName.VISIUM_LP_CYTASSIST,
```

Comment:
Is there a reason it defaults to VISIUM_LP_CYTASSIST rather that the XL version

Reply:
No, just selected the first in list so that there is an output already in display on loading page. Probably, will ask Katy if users have got any preference